.env

```diff
@@ -1,5 +1,7 @@
 NODE_LOCAL_PORT=3000
 DB_URL=mongodb+srv://PayingItForward:Super1Hard2Password3That4No5One6Nos7@payingitforward.uwxol.mongodb.net/paying-it-forward?retryWrites=true&w=majority
 TEST_DB_URL=mongodb+srv://PayingItForward:Super1Hard2Password3That4No5One6Nos7@payingitforward.uwxol.mongodb.net/paying-it-forward-test?retryWrites=true&w=majority
+REQUEST_MODEL_NAME=request
+ITEM_MODEL_NAME=item
 USER_MODEL_NAME=user
-RATING_MODEL_NAME=rating
\ No newline at end of file
+RATING_MODEL_NAME=rating
```

Comment:
Thanks for adding my changes too so we won't have conflicts 😅

Reply:
Yes, I noticed that after I pushed my first commit :laughing: